### PR TITLE
[fix](Nereids) fix insert into table with null literal default value

### PR DESCRIPTION
**Pull request by LiBinfeng-01**
Problem:
when use insert with default value null, it can not be insert successfully
Solved:
when column is allow to be null, it can be null in create table with null default value

### CODE REVIEW

**Review by morrySnow — APPROVED** [review]

File: regression-test/data/load_p0/insert/test_insert_default_value.out [hunk 1]
```diff
@@ -1,8 +1,15 @@
 -- This file is automatically generated. You should know what you did if you want to edit this
 -- !select1 --
-10	10000	10000000	92233720368547758	19223372036854775807	10.3	10.3
-10	10000	10000000	92233720368547758	19223372036854775807	10.3	10.3
+10	10000	10000000	92233720368547758	19223372036854775807	10.30	10.3
```
Comment (cambyzju) [inline regression-test/data/load_p0/insert/test_insert_default_value.out:3]: why this pr will change `10.3` to `10.30`

Comment (morrySnow) [reply regression-test/data/load_p0/insert/test_insert_default_value.out:3]: i think it is not this pr changed the result scale. the regression framework do not check precision and scale, so the test could pass even if scale has been changed

**Comment by LiBinfeng-01** [comment]
run buildall

**Comment by LiBinfeng-01** [comment]
run p0

**Review by feiniaofeiafei — APPROVED** [review]

**Review by morrySnow — APPROVED** [review]

**Pull request merged**